govcd/api_vcd_test.go

```diff
@@ -625,3 +634,61 @@ func TestVCDClient_Authenticate(t *testing.T) {
 		t.Fatalf("Error authenticating: %v", err)
 	}
 }
+
+func (vcd *TestVCD) createTestVapp(name string) (VApp, error) {
```

Comment:
Curious, why this test moved from vApp test to API test?

Reply:
This function is needed to create the initial vApp. 
If we leave it in `vapp_test.go`, every time we run a test we need to include vApp, even if we don't need it. With this change, we can isolate tests better. When vApp is not requested, the vApp creation gets skipped and the tests are faster.